frontend/src/i18n/hu.json

```diff
@@ -1,4 +1,12 @@
 {
+  "Accepted registration": "Sikeres regisztráció",
```

Comment:
Lehet ilyen szöveges kulcsokat is használni a fordításokhoz, de én inkább camel case-t használnék: acceptedRegistation